packager/packager.cc

```diff
@@ -1542,6 +1556,83 @@ unique_ptr<PackageInfoImpl> createAndPopulatePackageInfo(core::GlobalState &gs,
     return info;
 }
 
+// Metadata for Tarjan's algorithm
+struct ComputeSCCsMetadata {
```

Comment:
I'm getting a little bit lost in the GitHub review UI, but would you mind making sure that any internal functions here are inside `namespace {...}`, or if they're meant to be public that they have corresponding declarations in the `packager.h` header?

Reply:
Yup, everything from the start of the file to just before `validatePackage` is inside the the `namespace { ... }`.